README.md

```diff
@@ -65,6 +65,11 @@ https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/docs/app-sre/m
 to learn how to develop AppSRE dashboards for RHTAP. This repository serves as
 versioned storage for the [dashboard definitions](dashboards/) and nothing more.
 
+Dashboards are automatically deployed to [stage](https://grafana.stage.devshift.net) AppSRE Grafana when merged into the `main` branch.
+Deploying to [production](https://grafana.app-sre.devshift.net/) requires an update of a commit
```

Comment:
Do our metrics reach Grafana stage? We have different datasources for them in Grafana prod. Do we also have them sent to stage?

Reply:
Yes, there's a `rhtap-observatorium-stage` and `rhtap-observatorium-production` datasource in both Grafana instances.